app/Http/Requests/EditPostRequest.php

```diff
@@ -23,20 +24,30 @@ public function authorize()
      */
     public function rules()
     {
+        $postId = $this->route('id');
+        $tagMinRule = new TagMinRule(config('constants.MIN_TAG'));
+
         return [
-            'title' => 'required|min:6|unique:posts,title,' . $this->id,
-            'content' => 'required|min:6',
+            'title' => 'required|min:6|max:150|unique:posts,title,' . $postId,
+            'content' => 'required|max:5000',
+            'tags' => [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/EditPostRequest.php</br>**Line**: 33</br>Each line in an array declaration must end in a comma